load/loader.py

```diff
@@ -102,7 +102,16 @@ def load_voting_history(opts, conf):
         '-param:reportFileDir={}'.format(opts.input_file),
         '-param:reporterKey={}'.format(opts.reporter_key)
     ])
-
+    elif opts.state == 'ny':
+        fl_path = os.path.dirname(os.path.join(conf['data_path'],
```

Comment:
Actually fl_path is not used in either operation! Deleted that.

There is a difference between FL and NY in that NY is a single file, whereas FL is a directory of county files. That's why `reportFile` vs `reportFileDir` - we can make NY a default once we get the next state added in.